db.go

```diff
@@ -1320,6 +1336,9 @@ type Options struct {
 	// PageSize overrides the default OS page size.
 	PageSize int
 
+	// MaxSize sets the maximum size of the data file. 0 means no maximum.
```

Comment:
```suggestion
	// MaxSize sets the maximum size of the data file. 0 or a negative value means no limit.
```

Reply:
I made the comment "<=0" to match other comments like InitialMmapSize